hazelcast/src/test/java/com/hazelcast/nio/tcp/nonblocking/iobalancer/IOBalancerStressTest.java

```diff
@@ -74,7 +74,7 @@ public void testEachConnectionUseDifferentSelectorEventually() {
         IMap<Integer, Integer> map = instance1.getMap(randomMapName());
 
         long deadLine = System.currentTimeMillis() + TEST_DURATION_SECONDS * 1000;
-        for (int i = 0; System.currentTimeMillis() < deadLine; i++) {
+        for (int i = 0; System.currentTimeMillis() < deadLine && map.size() > 1000; i++) {
```

Comment:
If we keep the timing component, it still should be `map.size() < 1000` instead of `map.size() > 1000`, otherwise the loop will do nothing.